tests/pass-dep/tokio/sleep.rs

```diff
@@ -1,14 +1,17 @@
-//@compile-flags: -Zmiri-disable-isolation -Zmiri-permissive-provenance -Zmiri-backtrace=full
+//@compile-flags: -Zmiri-isolation-error=warn -Zmiri-permissive-provenance -Zmiri-backtrace=full
```

Comment:
What is the point of the `warn` setting here?

Reply:
I left it in while I was still developing & forgot about it since it’s no longer needed. It’s removed now.